src/lib/array.jl

```diff
@@ -165,12 +165,14 @@ end
 # needed for stateful functions.
 # See https://github.com/FluxML/Flux.jl/issues/1209
 # Should be generalized to abstract array, but reverse takes a dims keyword there
+
+# That lead to https://github.com/FluxML/Zygote.jl/issues/1374
+# ... and post-1.6 dims is not required to reverse arrays.
```

Comment:
How about amending the above comment instead? As noted it turned stale the moment we set min compat to 1.6.

Reply:
I've shrunk the comment. Not sure what the right policy is for `pmap`, but I haven't changed it.